Language_Parser/actionVerbs.py

```diff
@@ -257,13 +258,37 @@ def go(info):
 
 
 def look(info):
-    """"""
-    pass
+    """
+    Action function prints the long description of the Player's current room.
+    """
+    print(info["Player"].getLocation().getLongDescription())
 
 
 def eat(info):
-    """"""
-    pass
+    """
+    Action function prints response to the Player attempting to eat various
+    Items or Room features.
+    """
+    if len(info["Items"]) == 0:
+        print("You're thinking about eating something, but what?")
+        return
+    # Get target name that the player wants to eat from input
+    eatTarget = info["Items"][0]
+    # Look for item with target name in player inventory and current room
+    inventoryList = info["Player"].getInventory()
+    roomItemList = info["Player"].getLocation().getItems()
+    allItems = inventoryList + roomItemList
```

Comment:
Currently this doesn't work for the 3 polaroid pictures, because their names are "polaroid1", "polaroid2", and "polaroid3". I wrote a helper function to handle this when I wrote "drop()" and I think you could use it here. If you add these two lines after "allItems = inventoryList + roomItemList" then it will be able to handle the user trying to eat the polaroids:

    if eatTarget == "polaroid":
        eatTarget = identifyPolaroid(info["Player"])